src/types.ts

```diff
@@ -16,4 +17,5 @@ export interface DetailedAsyncAPI {
 }
 
 export type Diagnostic = ISpectralDiagnostic;
-export type SchemaValidateResult = IFunctionResult;
+export type SchemaValidateError = IFunctionResult;
```

Comment:
I changed the name to be more explicit about what it is in fact... just an error.

Reply:
It's not always an error, because from spectral you can also return info or warning severity, by this spectral team called it `IFunctionResult` not `IFunctionError`. Please change to the previous one.

Reply:
Rollbacked the rename! thanks for the catch